lib/test.gi

```diff
@@ -347,6 +358,20 @@ end);
 ##  reached. If a <C>#@else</C> is present then the code after the <C>#@else</C>
 ##  is used if and only if <C>EXPR</C> evaluated to <K>false</K>. Finally,
 ##  once <C>#endif</C> is reached, evaluation continues normally.
+##  <P/>
+##  For example, the package <Package>recog</Package> 
+##  contains the test file <C>tst/slow/MatAn.tst</C> which ends with 
+##  <Log><![CDATA[
+##  gap> if not IsPackageMarkedForLoading("cvec","") then 
+##  >       ri := RECOG.TestGroup(g,false,Factorial(51)/2);;
+##  >    fi;
+##  ]]></Log>
+##  This could equally well be rewritten: 
+##  <Log><![CDATA[
+##  #@if  not IsPackageMarkedForLoading("cvec","") 
+##  gap>  ri := RECOG.TestGroup(g,false,Factorial(51)/2);;
+##  #@fi
```

Comment:
How about taking an example from the GAP test suite instead? Or at least inspired by it. E.g.
```
#@if GAPInfo.BytesPerVariable = 8
gap> [ 1 .. 2^60-1 ];
[ 1 .. 1152921504606846975 ]
gap> [ 1 .. 2^60 ];
Error, Range: <last> must be a small integer (not a large positive integer)
#@else
gap> [ 1 .. 2^28-1 ];
[ 1 .. 268435455 ]
gap> [ 1 .. 2^28 ];
Error, Range: <last> must be a small integer (not a large positive integer)
#@fi
```

Reply:
Have used a short section from testinstall/pperm.tst